File: Ejercicios de Semana Santa/ejercicios de semana santa php/Two php/Library/Usuario.php
<?php
class Usuario {
    private $nombre;
    private $apellido;
    private $identificacion;
    private $peso;
    private $estatura;

    public function __construct($nombre, $apellido, $identificacion, $peso, $estatura) {
        $this->nombre = $nombre;
        $this->apellido = $apellido;
        $this->identificacion = $identificacion;
        $this->peso = $peso;
        $this->estatura = $estatura;
    }

    public function getNombreCompleto() {
        return $this->nombre . " " . $this->apellido;
    }

    public function getIdentificacion() {
        return $this->identificacion;
    }

    public function getPeso() {
        return $this->peso;
    }

    public function getEstatura() {
        return $this->estatura;
    }
}
?>

File: Prueba y error/ejercicios de semana santa java/two java/Library/Usuario.java
package Library;

public class Usuario {
    private String identificacion;
    private String nombres;
    private String apellidos;
    private double peso;
    private double estatura;
    private Plan plan;

    public Usuario(String identificacion, String nombres, String apellidos, double peso, double estatura, Plan plan) {
        this.identificacion = identificacion;
        this.nombres = nombres;
        this.apellidos = apellidos;
        this.peso = peso;
        this.estatura = estatura;
        this.plan = plan;
    }

    public String getIdentificacion() {
        return identificacion;
    }

    public String getNombres() {
        return nombres;
    }

    public String getApellidos() {
        return apellidos;
    }

    public double getPeso() {
        return peso;
    }

    public double getEstatura() {
        return estatura;
    }

    public Plan getPlan() {
        return plan;
    }

    // Método para mostrar detalles del usuario
    public String mostrarDetalles() {
        return "Usuario: " + nombres + " " + apellidos + "\n" +
               "Identificación: " + identificacion + "\n" +
               "Peso: " + peso + " kg\n" +
               "Estatura: " + estatura + " m\n" +
               plan.detalles();
    }
}
